backend/lambdas/generators/json_report/json_report/results/sbom.py

```diff
@@ -0,0 +1,22 @@
+from artemisdb.artemisdb.consts import PluginType
+from artemisdb.artemisdb.models import Scan
+from json_report.results.results import PLUGIN_RESULTS, PluginErrors
+
+
+def get_sbom(scan: Scan) -> PLUGIN_RESULTS:
+    """
+    Unify the output of sbom plugins. We deliberately omit the `details` property in the database
+    (since SBOM results are stored in s3), so this returns no scan results or summary
```

Comment:
This comment indicates why we return no `results` or `summary` but does not mention why `success` is hard-coded to `True`. I think I understand why, but would it be good to add this one since the others are explained?

Reply:
Will add it.

Basically, there is no SBOM scan state that would cause us to want `success` to be `False`. The SBOM scan results are not stored here (and are just informative anyway. I don't think one can "fail" an SBOM scan) and it would not make sense to fail a scan when an SBOM scan errors out.